### PR TITLE
added solidity in experimental

### PR DESCRIPTION
**Pull request by s-santillan**
### Security

- [x] Change has no security implications (otherwise, ping the security team)

### CODE REVIEW

File: docs/status.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ import MoreHelp from "/src/components/MoreHelp"
 | Ruby       |                            | Rust                       |
 | TypeScript |                            | YAML                       |
 | TSX        |                            | Generic (ERB, Jinja, etc.) |
-|            |                            |                            |
+|            |                            |  Solidity                          |
```
Comment (pabloest) [inline docs/status.md:30]: Could you place this below Rust and above YAML so they remain in alphabetical order?

**Comment by s-santillan** [comment]
Converted to draft, please hold for additional updates.

**Comment by s-santillan** [comment]
Updated the homepage table. That about covers it.

File: docs/index.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Semgrep supports 17+ languages.
 | Ruby       |                            | Rust                       |
 | TypeScript |                            | YAML                       |
```
Comment (pabloest) [inline docs/index.md:62]:
```suggestion
| TypeScript |                            | Solidity                   |
```

File: docs/index.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ Semgrep supports 17+ languages.
...
 | TSX        |                            | Generic (ERB, Jinja, etc.) |
```
Comment (pabloest) [inline docs/index.md:63]:
```suggestion
| TSX        |                            | YAML                       |
```

File: docs/index.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Semgrep supports 17+ languages.
...
-|            |                            |                            |
+|            |                            |  Solidity                        |
```
Comment (pabloest) [inline docs/index.md:64]:
```suggestion
|            |                            | Generic (ERB, Jinja, etc.) |
```

File: docs/status.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ import MoreHelp from "/src/components/MoreHelp"
 | Ruby       |                            | Rust                       |
 | TypeScript |                            | YAML                       |
```
Comment (pabloest) [inline docs/status.md:28]:
```suggestion
| TypeScript |                            | Solidity                   |
```

**Review by gitatb — APPROVED** [review]

File: docs/status.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ import MoreHelp from "/src/components/MoreHelp"
...
 | TSX        |                            | Generic (ERB, Jinja, etc.) |
```
Comment (pabloest) [inline docs/status.md:29]:
```suggestion
| TSX        |                            | YAML                       |
```

File: docs/status.md (hunk 1 above)
Comment (pabloest) [inline docs/status.md:30]:
```suggestion
|            |                            | Generic (ERB, Jinja, etc.) |
```

**Review by pabloest — APPROVED** [review]
Thank you, @s-santillan ! 

I left a suggestion to keep the alphabetical ordering, I _think_ it works out correctly.

File: docs/status.md (hunk 1 above)
Comment (s-santillan) [reply docs/status.md:30]: @pabloest Yup, working on it now

**Pull request merged**